templates/applicantForm.phtml

```diff
@@ -134,8 +134,13 @@
 
         <div id="assessment" class="collapse">
             <label for="assessmentDay">Assessment day</label>
+            <p>
+                <label for="pastAssessmentDays">
+                    <input type="checkbox" id="pastAssessmentDays" name="accept" value="yes">  Include past assessment days
+                </label>
+            </p>
+            <p>Current saved date: <?php echo $applicant->getAssessmentDay();?></p>
```

Comment:
Please could you only display this if there is a date currently set? Also, wondering if it's possible only to display it if the date is a past one?